### PR TITLE
Make the create-cluster script robustness

### PR DESCRIPTION
**Pull request by xiaobiaozhao**
1. Modify path error
2. Wait enough time

### CODE REVIEW

File: utils/create-cluster/create-cluster [hunk 1]
```diff
@@ -65,7 +65,7 @@ then
         sed -i.bak "s|port.*|port ${PORT}|g" ${conf_file} && rm ${conf_file}.bak
         sed -i.bak "s|dir.*|dir "node_${PORT}"|g" ${conf_file} && rm ${conf_file}.bak
         $BIN_PATH/kvrocks -c ${conf_file}
-        sleep 1
+        sleep 2 # wait enough time
```
Comment (tisonkun) [inline utils/create-cluster/create-cluster]: Can you share a bit more about how you evaluate this parameter? That is, why choose 2 (what is "enough")?

Comment (xiaobiaozhao) [reply utils/create-cluster/create-cluster]: Wait 1 second  will `Connection refused` on my test machine, 2 seconds is fine

Comment (xiaobiaozhao) [reply utils/create-cluster/create-cluster]:
```
./create-cluster start
Starting 30001
OK
OK
Starting 30002
OK
OK
Starting 30003
OK
OK
Starting 30004
Could not connect to Redis at 127.0.0.1:30004: Connection refused
Could not connect to Redis at 127.0.0.1:30004: Connection refused
Starting 30005
Could not connect to Redis at 127.0.0.1:30005: Connection refused
Could not connect to Redis at 127.0.0.1:30005: Connection refused
Starting 30006
OK
OK
```

Comment (git-hulk) [reply utils/create-cluster/create-cluster]: I think we can detect if the port is serving like the below:

```shell
until nc -z 127.0.0.1 3004; do
  echo "instance is not ready"
  sleep 1
done
```

Comment (xiaobiaozhao) [reply utils/create-cluster/create-cluster]: > 
a good idea

**Comment by xiaobiaozhao** [comment]
demo output
```
Starting 30001
instance is not ready wait 1/5s
OK
OK
Starting 30002
instance is not ready wait 1/5s
OK
OK
Starting 30003
instance is not ready wait 1/5s
instance is not ready wait 2/5s
OK
OK
Starting 30004
instance is not ready wait 1/5s
OK
OK
Starting 30005
instance is not ready wait 1/5s
instance is not ready wait 2/5s
OK
OK
Starting 30006
instance is not ready wait 1/5s
instance is not ready wait 2/5s
OK
OK
```

File: utils/create-cluster/create-cluster [hunk 2]
```diff
@@ -65,7 +65,18 @@ then
...
+        max_wait_times=5
+        until nc -z 127.0.0.1 ${PORT}; do
+            sleep 1
+            wait_times=$((wait_times+1))
+            echo "instance is not ready wait ${wait_times}/${max_wait_times}s"
+            if [[ ${wait_times} > 5 ]]
+            then
+                echo "instance is not readys, please check it:" ${PORT}
+                exit 1
+            fi
```
Comment (git-hulk) [inline utils/create-cluster/create-cluster]:
```suggestion
        tries = 0
        max_tries = 5
        until nc -z 127.0.0.1 ${PORT}; do
            sleep 1
            tries = $((tries+1))
            echo "instance port ${PORT} is not ready, will retry after 1 second"
            if [[ ${tries} > ${max_tries} ]]
            then
                echo "instance port ${PORT} reached max tries(${max_tries}), will exit"
                exit 1
            fi
```

**Review by git-hulk — APPROVED** [review]
LGTM

**Pull request merged**